packages/configs/default/index.json

```diff
@@ -68,8 +68,6 @@
   "resolvers": ["@parcel/resolver-default"],
   "reporters": [
     "@parcel/reporter-cli",
-    "@parcel/reporter-dev-server",
-    "@parcel/reporter-bundle-analyzer",
-    "@parcel/reporter-bundle-buddy"
```

Comment:
Removed these temporarily. The ideal way to run these is via a `--reporter` CLI option (which we have yet to add).

Reply:
Including the dev server? Sure?

Reply:
Dev server is still there...